bundles/test/org.eclipse.smarthome.magic/src/main/java/org/eclipse/smarthome/magic/binding/internal/http/MagicHttpRessource.java

```diff
@@ -0,0 +1,95 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.magic.binding.internal.http;
+
+import java.io.IOException;
+import java.net.URL;
+import java.util.Map;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.osgi.framework.Bundle;
+import org.osgi.framework.BundleContext;
+import org.osgi.service.component.annotations.Activate;
+import org.osgi.service.component.annotations.Component;
+import org.osgi.service.component.annotations.Deactivate;
+import org.osgi.service.component.annotations.Reference;
+import org.osgi.service.http.HttpContext;
+import org.osgi.service.http.HttpService;
+import org.osgi.service.http.NamespaceException;
+
+/**
+ * Registers itself under "/" and provides links to other web UIs
+ *
+ * @author Stefan Triller - initial contribution
+ *
+ */
+@Component(immediate = true)
+public class MagicHttpRessource {
+    /** the root path of this web application */
+    private static final String WEBAPP_ALIAS = "/";
+
+    protected HttpService httpService;
+
+    @Activate
+    protected void activate(Map<String, Object> configProps, BundleContext bundleContext) throws NamespaceException {
+        httpService.registerResources(WEBAPP_ALIAS, WEBAPP_ALIAS, new MagicHttpContext(bundleContext.getBundle()));
```

Comment:
Isn't it a little bit confusing to use the variable WEBAPP_ALIAS for the name (so, the resource prefix)?